mflowgen/common/custom-power-chip/outputs/globalnetconnect.tcl

```diff
@@ -9,3 +9,11 @@ globalNetConnect VSS -type pgpin -pin VBB -inst *
 # I/O power nets
 globalNetConnect VDDPST -type pgpin -pin VDDPST -inst *
 globalNetConnect VSS    -type pgpin -pin VSSPST -inst * 
+
+# Dragonphy power nets
+globalNetConnect VDD -type pgpin -pin DVDD -inst iphy -override
+globalNetConnect VSS -type pgpin -pin DVSS -inst iphy -override
+globalNetConnect AVDD -type pgpin -pin AVDD -inst iphy -override
+globalNetConnect AVSS -type pgpin -pin AVSS -inst iphy -override
+globalNetConnect CVDD -type pgpin -pin CVDD -inst iphy -override
+globalNetConnect CVSS -type pgpin -pin CVSS -inst iphy -override
```

Comment:
Here they are in globalnetconnect.tcl. Or am I looking at an old file somehow?

Reply:
The issue is here: https://github.com/StanfordAHA/garnet/blob/2aba45dcda61f26a6009287a41bd48ff26b5206f/mflowgen/common/fc-netlist-fixing/outputs/netlist-fixing.tcl#L10

The AVDD, AVSS, CVDD, and CVSS nets do not exist until the netlist fixing step. These `globalNetConnect` commands actually fail in the power step for this reason. The best solution would probably be to create these nets during the power step, or to remove those globalNetConnect commands from globalnetconnect.tcl.

Reply:
Understood. Carry on!